### DISCUSSION TITLE
Remove depth dimension in trajectory output for 2D advection

### DISCUSSION THREAD

**Discussion by sruehs** [Parcels-code/Parcels#1149] [General]
It appears that per default parcels records and outputs the particle depths even for pure 2D advection (e.g., when applying the AdvectionRK4 Kernel). In my opinion, this unnecessarily enlarges the output file size. Maybe worth changing the default settings or providing an example how to tweak one's custom code to circumvent this issue?

**Comment by erikvansebille** [comment]
I dug a bit into the code, and we do have a function for this already (implemented by @CKehl): 
```python
pset.set_variable_write_status(var, write_status)
```

So to write only the starting depth of the particles (not at every timestep), you could do
```python
pset.set_variable_write_status('depth', 'once')
```

And to not write at all, you could do
```python
pset.set_variable_write_status('depth', False)
```

**Reply by sruehs** [reply]
Thanks for the fast reply. Sounds good - will give it a try!
